doc/manual/manual.bib

```diff
@@ -611,6 +611,16 @@ @Article{HGSLM03
   pages =	 {15--30}
 }
 
+@Article{CSG12,
+  title={A comparison of numerical surface topography calculations in geodynamic modelling: An evaluation of the ‘sticky air’method},
```

Comment:
Use single ticks (') around sticky air, rather than the Unicode equivalent. Space missing before "method".